edge/pkg/edgehub/edgehub.go

```diff
@@ -13,8 +14,7 @@ import (
 	"github.com/kubeedge/kubeedge/edge/pkg/edgehub/certificate"
 	"github.com/kubeedge/kubeedge/edge/pkg/edgehub/clients"
 	"github.com/kubeedge/kubeedge/edge/pkg/edgehub/config"
-	// register Upgrade handler
-	_ "github.com/kubeedge/kubeedge/edge/pkg/edgehub/upgrade"
+	_ "github.com/kubeedge/kubeedge/edge/pkg/edgehub/upgrade" // register Upgrade handler
```

Comment:
edge/pkg/edgehub/edgehub.go:17: File is not `gci`-ed with --skip-generated -s standard,default (gci)

You can recover changes to pass lint

Reply:
Yes this is fixed. This is not showing error when running `make lint` locally.